arkouda/strings.py

```diff
@@ -754,3 +754,16 @@ def save(self, prefix_path : str, dataset : str='strings_array',
         """       
         self.bytes.save(prefix_path=prefix_path, 
                                     dataset='{}/values'.format(dataset), mode=mode)
+
+    def register(self, user_defined_name : str) -> 'Strings':
+        return Strings(self.offsets.register(user_defined_name+'_offsets'),
```

Comment:
this needs `self` and the class construction... `@classmethod`  doesn't seem to fit the bill, suggestions?

Reply:
@mhmerrill sure, you could do that.